AboutUs/aboutUs.js

```diff
@@ -0,0 +1,19 @@
+function onFormSubmit() {
+    const username = document.getElementById("username").value
+    const password = document.getElementById("password").value
+    const date = document.getElementById("date_id").value
+    const checkbox = document.getElementById("marital_status").checked
+    const maleRadio = document.getElementById("male_radio").checked
+    const femaleRadio = document.getElementById("female_radio").checked
```

Comment:
I think there should be a better way to find what radio option is selected.

Reply:
I haven't find any other method